src/platform/forms/tests/save-in-progress/01-sip-autosave.e2e.spec.js

```diff
@@ -1,11 +1,12 @@
 const moment = require('moment');
-const E2eHelpers = require('../../../testing/e2e/helpers');
-const Timeouts = require('../../../testing/e2e/timeouts.js');
-const HcaHelpers = require('../../../../applications/hca/tests/hca-helpers.js');
+const E2eHelpers = require('platform/testing/e2e/helpers');
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+const HcaHelpers = require('applications/hca/tests/hca-helpers.js');
 
 module.exports = E2eHelpers.createE2eTest(client => {
   const url = `${E2eHelpers.baseUrl}/health-care/apply/application`;
   const token = HcaHelpers.initSaveInProgressMock(url, client);
+  HcaHelpers.initEnrollmentStatusMock(token);
```

Comment:
This test is using the HCA to test autosave functionality. To make this test pass locally (and in CI when the HCA changes are pushed to prod) we need to mock the `enrollment_status` endpoint that the introduction page will depend on.